osu.Game/Screens/SelectV2/PanelSetBackground.cs

```diff
@@ -64,10 +70,16 @@ protected override void Update()
         }
 
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(OverlayColourProvider colourProvider)
         {
             InternalChildren = new Drawable[]
             {
+                new Box
+                {
+                    Depth = 1,
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colourProvider.Background4,
+                },
```

Comment:
Previously this box was present as `Panel.backgroundGradient` behind the beatmap background in set/standalone panels, but now that it has been removed from `Panel`, I have brought it back here inside the background class.

It's necessary to have when a beatmap background is not present, and also to hide the white/coloured background from flashing the user.

Background not present:

<img width="1492" height="625" alt="Image" src="https://github.com/user-attachments/assets/284f8a6e-b65f-4f4c-bf8e-6baf1c690dbe" />

How the white/coloured background can flash the user when this is not present:

https://github.com/user-attachments/assets/10f4fb4c-8b96-48cd-8b0d-e21160ae0ddb

Reply:
Right, https://github.com/ppy/osu/pull/34312/commits/86ed09c76ffa4ac2ef95aabbad08b8f7718ccdb2.